install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -42,14 +42,27 @@ $ openstack quota set --secgroups 30 --secgroup-rules 200 --ports 200 <project>
 [[osp_accounts_kuryr]]
 ==== Extra steps for Kuryr SDN
 
-If Kuryr SDN is enabled, and particularly if namespace isolation is used,
-increase the next quotas need to be raised: at least 100 security groups,
-500 security group rules, and 500 ports.
-----
-$ openstack quota set --secgroups 100 --secgroup-rules 500 --ports 500 <project>
-----
+If Kuryr SDN is enabled, especially if you are using namespace isolation,
+increase your project's quotas to:
+
+* 300 security groups (one for each namespace plus one for each load balancer)
+* 150 networks (one for each namespace)
+* 150 subnets (one for each namespace)
+* 500 security group rules
+* 500 ports (one port per Pod and additional ports for pools to speed up Pod
+  creation)
+
+[NOTE]
+====
+This is not a global recommendation. Adjust your quotas based on your needs.
+====
+
+If namespace isolation is enabled, each namespace is given a new network and
```

Comment:
You changed an earlier reference to _if namespace isolation is used_ to remove the passive. Should we do the same here? _If you are using namespace isolation_